sockeye/encoder.py

```diff
@@ -703,13 +777,15 @@ def __init__(self, encoders: List[Encoder], dtype: str = C.DTYPE_FP32) -> None:
     def encode(self,
                data: mx.sym.Symbol,
                data_length: mx.sym.Symbol,
-               seq_len: int) -> Tuple[mx.sym.Symbol, mx.sym.Symbol, int]:
+               seq_len: int,
+               att_dict: Optional[Dict[str, mx.sym.Symbol]]=None) -> Tuple[mx.sym.Symbol, mx.sym.Symbol, int]:
```

Comment:
spacing around "=None", also in the rest of the file.

Reply:
fixed